src/System.Net.NameResolution/src/System.Net.NameResolution.csproj

```diff
@@ -10,6 +10,9 @@
     <OutputType>Library</OutputType>
     <ProjectGuid>{1714448C-211E-48C1-8B7E-4EE667D336A1}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net46'">true</IsPartialFacadeAssembly>
+    <ResourcesSourceOutputDirectory Condition="'$(TargetGroup)' == 'net46'">None</ResourcesSourceOutputDirectory>
```

Comment:
Sorry no I was confusing it with another similar thing I did recently. However we probably should add one for this scenario :)